milatools/cli/commands.py

```diff
@@ -1181,7 +1195,7 @@ def _parse_lfs_quota_output(
     return (used_gb, max_gb), (used_files, max_files)
 
 
-def check_disk_quota(remote: Remote | RemoteV2) -> None:
+def check_disk_quota(remote: RemoteV1 | RemoteV2) -> None:
```

Comment:
This was probably changed during the rebase. `check_disk_quota` is currently (on the `master` branch) only ever called with a `RemoteV1`. This will change in #105 .